cc/constants.h

```diff
@@ -34,7 +34,11 @@ constexpr float kDefaultKomi = 7.5;
 
 constexpr float kDirichletAlpha = 0.03f * 361 / (kN * kN);
 
-static constexpr float kPuct = 2.50;
+// static constexpr float kPuct = 2.50;
+static constexpr float kUct_base = 19652;
+
+// We keep our -1 to 1 scale in the tree.
+static constexpr float kUct_init = 2.50;
```

Comment:
why not leave this unchanged and multiply the whole result by 2?

Reply:
done.